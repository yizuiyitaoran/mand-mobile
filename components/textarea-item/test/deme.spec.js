import Demo0 from './cases/demo0'
import Demo1 from './cases/demo1'
import Demo2 from './cases/demo2'
import {renderToString} from '@vue/server-test-utils'

describe('TextareaItem - Demo', () => {
  test('Basic', () => {
    expect(renderToString(Demo0)).toMatchSnapshot()
  })
  test('Error Tip', () => {
    expect(renderToString(Demo1)).toMatchSnapshot()
  })
  test('Slot', () => {
    expect(renderToString(Demo2)).toMatchSnapshot()
  })
})
