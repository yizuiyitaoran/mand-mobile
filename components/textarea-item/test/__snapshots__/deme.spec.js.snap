// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextareaItem - Demo Basic 1`] = `
<div class="md-example-child md-example-child-textarea md-example-child-textarea-0">
  <fieldset class="md-field">
    <!---->
    <div class="md-field-content">
      <div class="md-field-item md-textarea-item example is-solid is-browser">
        <div class="md-field-item-content"><label class="md-field-item-title">普通</label>
          <!---->
          <div class="md-field-item-control"><textarea maxlength="" placeholder="普通文本域" rows="3" class="md-textarea-item__textarea"></textarea> </div>
          <!---->
        </div>
        <!---->
      </div>
      <div class="md-field-item md-textarea-item example is-solid is-browser">
        <div class="md-field-item-content"><label class="md-field-item-title">自动适应</label>
          <!---->
          <div class="md-field-item-control"><textarea maxlength="" placeholder="最大高度150px" rows="1" class="md-textarea-item__textarea"></textarea> </div>
          <!---->
        </div>
        <!---->
      </div>
      <div class="md-field-item md-textarea-item example is-solid is-browser">
        <div class="md-field-item-content"><label class="md-field-item-title">删除图标</label>
          <!---->
          <div class="md-field-item-control"><textarea maxlength="" placeholder="输入出现删除图标" rows="1" class="md-textarea-item__textarea"></textarea> </div>
          <div class="md-field-item-right">
            <div class="md-textarea-item__clear" style="display:none;"><i class="md-icon icon-font md-icon-clear clear md" style="color:;"></i></div>
            <!---->
          </div>
        </div>
        <!---->
      </div>
      <div class="md-field-item md-textarea-item example is-solid is-browser is-disabled">
        <div class="md-field-item-content"><label class="md-field-item-title">禁用</label>
          <!---->
          <div class="md-field-item-control"><textarea disabled="disabled" maxlength="" placeholder="500字以内" rows="3" class="md-textarea-item__textarea">禁用文本域, 理赔报案描述, 理赔报案描述, 理赔报案描述, 理赔报案描述, 理赔报案描述, 理赔报案描述</textarea> </div>
          <!---->
        </div>
        <!---->
      </div>
      <div class="md-field-item md-textarea-item example is-solid is-browser">
        <div class="md-field-item-content"><label class="md-field-item-title">只读</label>
          <!---->
          <div class="md-field-item-control"><textarea readonly="readonly" maxlength="" placeholder="500字以内" rows="3" class="md-textarea-item__textarea">只读文本域, 理赔报案描述, 理赔报案描述, 理赔报案描述, 理赔报案描述, 理赔报案描述, 理赔报案描述</textarea> </div>
          <!---->
        </div>
        <!---->
      </div>
    </div>
    <!---->
  </fieldset>
</div>
`;

exports[`TextareaItem - Demo Error Tip 1`] = `
<div class="md-example-child md-example-child-textarea md-example-child-textarea-0">
  <fieldset class="md-field">
    <!---->
    <div class="md-field-content">
      <div class="md-field-item md-textarea-item example is-solid is-browser is-error">
        <div class="md-field-item-content"><label class="md-field-item-title">错误提示</label>
          <!---->
          <div class="md-field-item-control"><textarea maxlength="" placeholder="" rows="3" class="md-textarea-item__textarea">申请理赔xxx</textarea> </div>
          <!---->
        </div>
        <div class="md-field-item-children">
          <div class="md-textarea-item-msg">
            <p>输入信息有误!</p>
          </div>
        </div>
      </div>
    </div>
    <!---->
  </fieldset>
</div>
`;

exports[`TextareaItem - Demo Slot 1`] = `
<div class="md-example-child md-example-child-textarea md-example-child-textarea-2">
  <fieldset class="md-field">
    <!---->
    <div class="md-field-content">
      <div class="md-field-item md-textarea-item demo2 is-solid is-browser">
        <div class="md-field-item-content">
          <!---->
          <!---->
          <div class="md-field-item-control"><textarea maxlength="100" placeholder="描述信息在100字以内" rows="3" class="md-textarea-item__textarea"></textarea>
            <p class="demo2__footer"><span></span> <span class="demo2__footer-right">页脚文案</span></p>
          </div>
          <!---->
        </div>
        <!---->
      </div>
    </div>
    <!---->
  </fieldset>
</div>
`;
